### ISSUE TITLE
Mail not being sent

### ISSUE THREAD

**Issue by justintolerable** [simple-login/app#323]
I know it's a fairly common issue, but I'm having trouble getting emails sent out from the server.  I've chatted briefly with Linode, who have lifted port 25 restrictions, but still no luck.  It just times out when trying to contact the recipient email address.  I manually activated my account in the database to create a test alias.  The dns records all check out just fine and I can access the web app.  Here's part of the mail.log (the rest is largely similar).

> Nov  1 19:38:46 localhost postfix/postfix-script[1872]: starting the Postfix mail system
> Nov  1 19:38:46 localhost postfix/master[1874]: daemon started -- version 3.3.0, configuration /etc/postfix
> Nov  1 19:42:55 localhost postfix/postfix-script[2944]: stopping the Postfix mail system
> Nov  1 19:42:55 localhost postfix/master[1874]: terminating on signal 15
> Nov  1 19:42:56 localhost postfix/postfix-script[3101]: starting the Postfix mail system
> Nov  1 19:42:56 localhost postfix/master[3103]: daemon started -- version 3.3.0, configuration /etc/postfix
> Nov  1 19:51:30 localhost postfix/smtpd[4756]: connect from unknown[240.0.0.3]
> Nov  1 19:51:30 localhost postfix/smtpd[4756]: 1B85A197E94: client=unknown[240.0.0.3]
> Nov  1 19:51:30 localhost postfix/cleanup[4765]: 1B85A197E94: message-id=<160426029005.8.12619477958150527081@75d504410f97>
> Nov  1 19:51:30 localhost postfix/qmgr[3105]: 1B85A197E94: from=<support@hidden.click>, size=22198, nrcpt=1 (queue active)
> Nov  1 19:51:30 localhost postfix/smtpd[4756]: disconnect from unknown[240.0.0.3] ehlo=1 mail=1 rcpt=1 data=1 commands=4
> Nov  1 19:52:00 localhost postfix/smtp[4766]: connect to mail.protonmail.ch[185.70.41.101]:25: Connection timed out
> Nov  1 19:52:00 localhost postfix/smtp[4766]: 1B85A197E94: to=<[recipient email address removed for privacy]>, relay=none, delay=30, delays=0.03/0.02/30/0, dsn=4.4.1, status=deferred (connect to mail.protonmail.ch[185.70.41.101]:25: Connection timed out)
> Nov  1 19:54:57 localhost postfix/smtpd[4816]: connect from mail-lj1-f179.google.com[209.85.208.179]
> Nov  1 19:54:58 localhost postfix/smtpd[4816]: 7EB7819761B: client=mail-lj1-f179.google.com[209.85.208.179]
> Nov  1 19:54:58 localhost postfix/cleanup[4823]: 7EB7819761B: message-id=<CACRgKw6-G8h+2yXpX++nmq6ChhDdTFPBcHewYpkuzt12Jmu_cg@mail.gmail.com>
> Nov  1 19:54:58 localhost postfix/qmgr[3105]: 7EB7819761B: from=<[sender email address removed for privacy]>, size=5500, nrcpt=1 (queue active)
> Nov  1 19:54:58 localhost postfix/smtpd[4825]: connect from unknown[240.0.0.4]
> Nov  1 19:54:58 localhost postfix/smtpd[4816]: disconnect from mail-lj1-f179.google.com[209.85.208.179] ehlo=2 starttls=1 mail=1 rcpt=1 data=1 quit=1 commands=7
> Nov  1 19:54:59 localhost postfix/smtpd[4825]: 05DB5197E9B: client=unknown[240.0.0.4]
> Nov  1 19:54:59 localhost postfix/cleanup[4823]: 05DB5197E9B: message-id=<CACRgKw6-G8h+2yXpX++nmq6ChhDdTFPBcHewYpkuzt12Jmu_cg@mail.gmail.com>
> Nov  1 19:54:59 localhost postfix/qmgr[3105]: 05DB5197E9B: from=<reply+sctnzgoigktjxpvcwrqqzygezmmzwp@hidden.click>, size=5650, nrcpt=1 (queue active)
> Nov  1 19:54:59 localhost postfix/smtpd[4825]: disconnect from unknown[240.0.0.4] ehlo=1 mail=1 rcpt=1 data=1 commands=4
> Nov  1 19:54:59 localhost postfix/smtp[4824]: 7EB7819761B: to=<[alias removed for privacy]@hidden.click>, relay=127.0.0.1[127.0.0.1]:20381, delay=0.87, delays=0.44/0.02/0/0.4, dsn=2.0.0, status=sent (250 Message accepted for delivery)
> Nov  1 19:54:59 localhost postfix/qmgr[3105]: 7EB7819761B: removed
> Nov  1 19:55:29 localhost postfix/smtp[4824]: connect to mail.protonmail.ch[185.70.41.101]:25: Connection timed out
> Nov  1 19:55:29 localhost postfix/smtp[4824]: 05DB5197E9B: to=<[recipient email address removed for privacy]>, relay=none, delay=30, delays=0.01/0/30/0, dsn=4.4.1, status=deferred (connect to mail.protonmail.ch[185.70.41.101]:25: Connection timed out)
> Nov  1 19:57:56 localhost postfix/qmgr[3105]: 1B85A197E94: from=<support@hidden.click>, size=22198, nrcpt=1 (queue active)
> Nov  1 19:58:18 localhost postfix/anvil[4818]: statistics: max connection rate 1/60s for (smtp:209.85.208.179) at Nov  1 19:54:57
> Nov  1 19:58:18 localhost postfix/anvil[4818]: statistics: max connection count 1 for (smtp:209.85.208.179) at Nov  1 19:54:57
> Nov  1 19:58:18 localhost postfix/anvil[4818]: statistics: max cache size 1 at Nov  1 19:54:57
> Nov  1 19:58:26 localhost postfix/smtp[4856]: connect to mail.protonmail.ch[185.70.41.101]:25: Connection timed out

Any idea what I'm missing?  I'm fairly new to this and I'm really starting to struggle.  I've redone the process a few times, following along as closely as I can, but just can't figure out this part.

**Comment by justintolerable** [comment]
The issue, was predictably, that the outgoing 25 was still blocked by linode.  They had told me it was unblocked, but then when they checked again later, they discovered they had made an error.  Putting this here for others to see.
